README.md

```diff
@@ -320,6 +322,85 @@ i18n.offChangeLocale(fn);
 i18n.onceChangeLocale(fn);
 ```
 
+## Pluralization
+
+With this package, you can translate with pluralization. To do that you need to use the pipe separator `|` in your locale translations, like in the example below:
+
+```yml
+_locale: 'en'
+phone: 'zero phones | one phone | two phones | {count} phones'
```

Comment:
```suggestion
phone: 'zero phones | one phone | two phones | {$count} phones'
```

Maybe we can use `$count` instead. We use this convention in the [string interpolation](https://github.com/vazco/meteor-universe-i18n#string-interpolation).

What do you think about using pluralization in interpolation?

Reply:
A good point to keep the naming consistent. As for the pluralization in interpolation, I think it's a good idea but maybe let's handle it in a separate PR?